diff --git a/src/messageArea.js b/src/messageArea.js
index 307602c6..518f1b39 100644
--- a/src/messageArea.js
+++ b/src/messageArea.js
@@ -282,6 +282,7 @@ export function messageArea (dom, kb, subject, messageStore, options) {
     }
 
     const td1 = dom.createElement('td')
+    td1.setAttribute('style','overflow-wrap:normal');
     tr.appendChild(td1)
     creatorAndDate(td1, creator, UI.widgets.shortDate(dateString), message)
 
@@ -342,7 +343,9 @@ export function messageArea (dom, kb, subject, messageStore, options) {
   messageTable = dom.createElement('table')
   messageTable.fresh = false
   div.appendChild(messageTable)
-  messageTable.setAttribute('style', 'width: 100%;') // fill that div!
+  messageTable.setAttribute('style', 'width: 100%;overflow-wrap: anywhere;'); 
+
+
 
   const tr = newMessageForm()
   if (newestFirst) {
diff --git a/src/widgets/forms.js b/src/widgets/forms.js
index 80f1a27b..ade6dcf4 100644
--- a/src/widgets/forms.js
+++ b/src/widgets/forms.js
@@ -270,7 +270,8 @@ field[ns.ui('Multiple').uri] = function (
       await saveListThenRefresh()
     } else {
       // eslint-disable-next-line multiline-ternary
-      const toBeInserted = reverse ? [$rdf.st(object, property, subject, dataDoc)] : [$rdf.st(subject, property, object, dataDoc)]
+      let toBeInserted = reverse ? [$rdf.st(object, property, subject, dataDoc)] : [$rdf.st(subject, property, object, dataDoc)]
+      if(category) toBeInserted.push($rdf.st(object,ns.rdf("type"),category,dataDoc))
       try {
         await kb.updater.update([], toBeInserted)
       } catch (err) {
@@ -424,6 +425,7 @@ field[ns.ui('Multiple').uri] = function (
   const ordered = orderedNode ? $rdf.Node.toJS(orderedNode) : false
 
   const property = kb.any(form, ui('property'))
+  const category = kb.any(form, ui('category'))
   const reverse = kb.anyJS(form, ui('reverse'), null, formDoc)
   if (!property) {
     box.appendChild(
