diff --git a/sourcePane.js b/sourcePane.js
index 4ea25c2..c82c9a2 100644
--- a/sourcePane.js
+++ b/sourcePane.js
@@ -197,7 +197,8 @@ module.exports = {
           if (!happy(response, 'PUT')) return
           /// @@ show edited: make save button disabled util edited again.
           setEditable()
-        })
+          eTag = response.headers.get('etag')
+	})
         .catch(function (err) {
           div.appendChild(
             UI.widgets.errorMessageBlock(dom, 'Error saving back: ' + err)
